### ISSUE TITLE
Generating new app with version 1.1 with `--no-html` option leads to compile errors

### ISSUE THREAD

**Issue by jaischeema** [phoenixframework/phoenix#1417]
I generated a new app using `--no-brunch --no-html` options and got this error while trying to start the server.

```elixir
== Compilation error on file web/views/error_view.ex ==
** (CompileError) web/views/error_view.ex:2: module Test.ErrorHelpers is not loaded and could not be found
    expanding macro: Test.Web.__using__/1
    web/views/error_view.ex:2: Test.ErrorView (module)
    (elixir) expanding macro: Kernel.use/2
    web/views/error_view.ex:2: Test.ErrorView (module)
    (elixir) lib/kernel/parallel_compiler.ex:100: anonymous fn/4 in Kernel.ParallelCompiler.spawn_compilers/8
```

To make sure it was only happening with `--no-html`, I created another test project with just that option and got the same result. To reproduce this:

1) Create new project `mix phoenix.new test_compile_error --no-html`
2) `cd test_compile_error`
3) `mix phoenix.server`

**Comment by chrismccord** [comment]
This is fixed on master and will be included in a 1.1.1 release tomorrow. Thanks!

**Comment by jaischeema** [comment]
Awesome, great work :+1:
